locales/en/cos-ui.json

```diff
@@ -211,18 +210,18 @@
   "reviewJSONStepDescription": "Review the configuration properties for your connector instance.\nThe JSON shown only includes the properties needed to configure the connector instance. Properties where default values are used are not included in the JSON.",
   "reviewStepDescription": "Review the configuration properties for your connector instance.",
   "rhocSubscriptionURL": "https://www.redhat.com/en/technologies/cloud-computing/openshift/connectors",
-  "rosaAddOnMsg": "Install the Connectors add-on on a ROSA cluster to create a namespace.",
+  "rosaAddOnMsg": "To create this namespace, you must install the Connectors add-on on a ROSA cluster to create a namespace.",
   "running": "Running",
   "save": "Save",
   "secretKey": "Secret Key",
   "secretValue": "**************************",
   "selectOcmCluster": "Select OCM cluster",
   "serviceAccount": "Service account",
   "serviceAccountAlertMsg": "Make a copy of the client ID and secret to store in a safe place. The client secret won't appear again after closing this screen.",
-  "serviceAccountDescText": "A service account enables the Connectors instance to authenticate with the Kafka instance. Provide credentials of a service account that has access to the Kafka instance, or create a new one. Define the service account's permissions from the Access tab of the Kafka instance so that source connectors can produce to, and sink connectors can consume from, topics.",
+  "serviceAccountDescText": "<p>A service account enables the Connectors instance to authenticate with the Kafka instance. This authentication allows  connectors to produce messages to (or consume messages from) the topics in the Kafka instance.</p> <p>Provide the credentials of a service account that has access to the Kafka instance, or create a new service account.</p><p> To define access to a Kafka instance, go to the Access tab for the Kafka instance.</p>",
```

Comment:
I think since this has become such a long string it'd be okay to break this up into separate parts, that way you can also move the markup out of here and into the component.